platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegionDefinition.java

```diff
@@ -12,6 +12,51 @@
 @Keep
 public interface OfflineRegionDefinition {
 
+  /**
+   * Gets the bounds of the OfflineRegion.
+   *
+   * @return the bounds of the OfflineRegion.
+   */
   LatLngBounds getBounds();
 
+  /**
+   * Returns the map style url of the OfflineRegion.
+   *
+   * @return The URL of the map style.
+   * @since 7.0.0
```

Comment:
I believe we can skip `@since` annotation, we are not using anywhere in the project.